src/Operations.jl

```diff
@@ -1782,4 +1797,114 @@ function status(env::EnvCache, pkgs::Vector{PackageSpec}=PackageSpec[];
     end
 end
 
+function check_force_latest_compatible_version(ctx::Types.Context;
+                                               target_name = nothing,
+                                               allow_earlier_backwards_compatible_versions::Bool = true)
+    direct_deps = load_direct_deps(ctx.env)
+    direct_deps_uuids = [dep.uuid for dep in direct_deps]
+    nonstdlib_direct_deps_uuids = filter(!is_stdlib, direct_deps_uuids)
+    result = check_force_latest_compatible_version(
+        ctx,
+        nonstdlib_direct_deps_uuids;
+        target_name,
+        allow_earlier_backwards_compatible_versions,
+    )
+    return result
+end
+
+function check_force_latest_compatible_version(ctx::Types.Context,
+                                               uuid_list::AbstractVector{Base.UUID};
+                                               target_name = nothing,
+                                               allow_earlier_backwards_compatible_versions::Bool = true)
+    isempty(uuid_list) && return true
+    results = check_force_latest_compatible_version.(
+        Ref(ctx),
+        uuid_list;
+        target_name,
+        allow_earlier_backwards_compatible_versions,
+    )
+    return all(results)
+end
```

Comment:
Perhaps simpler to merge these two functions:

```suggestion
function check_force_latest_compatible_version(ctx::Types.Context;
                                               target_name = nothing,
                                               allow_earlier_backwards_compatible_versions::Bool = true)
    direct_deps = load_direct_deps(ctx.env)
    direct_deps_uuids = [dep.uuid for dep in direct_deps]
    uuid_list = filter(!is_stdlib, direct_deps_uuids)
    isempty(uuid_list) && return true
    results = check_force_latest_compatible_version.(
        Ref(ctx),
        uuid_list;
        target_name,
        allow_earlier_backwards_compatible_versions,
    )
    return all(results)
end
```